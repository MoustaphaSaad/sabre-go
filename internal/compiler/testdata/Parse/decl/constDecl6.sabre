const (
	Sunday = iota
	Monday
	Tuesday
	Wednesday
	Thursday
	Friday
	Partyday
)