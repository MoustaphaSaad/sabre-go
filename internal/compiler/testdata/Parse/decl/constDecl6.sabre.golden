(ConstDecl
  (ConstDeclSpec
    (IdentifierExpr IDENTIFIER(Sunday))
    =
    (IdentifierExpr IDENTIFIER(iota))
  )
  (ConstDeclSpec
    (IdentifierExpr IDENTIFIER(Monday))
  )
  (ConstDeclSpec
    (IdentifierExpr IDENTIFIER(Tuesday))
  )
  (ConstDeclSpec
    (IdentifierExpr IDENTIFIER(Wednesday))
  )
  (ConstDeclSpec
    (IdentifierExpr IDENTIFIER(Thursday))
  )
  (ConstDeclSpec
    (IdentifierExpr IDENTIFIER(Friday))
  )
  (ConstDeclSpec
    (IdentifierExpr IDENTIFIER(Partyday))
  )
)